README.md

```diff
@@ -192,6 +192,7 @@ kubectl create secret generic realmsecret \
 helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
 
 helm install alfresco-incubator/alfresco-identity-service \
+--set keycloak.keycloak.extraArgs: "-Dkeycloak.import=/realm/realm.json"
```

Comment:
This is already set in `values.yaml`. I suggest we keep it in there so that command lines are as short as possible. Admins will know how to override it.

Reply:
I think this is necessary for other consumers of the identity service.

Reply:
An example is Activiti 7 being able to bring in their `springboot` realm until they switch over to the `alfresco` realm.

We might want to think about changing the flow of the documentation a bit though.

If you just follow the commands you don't realize you might need to customize the realm until after you've deployed, which would require you to delete the release, etc.

We should consider adding a note that alerts people that the chart will install the `alfresco` realm by default and point them to jump to the custom realm section if they need to.